docs/EN_US/CMakeLists.txt

```diff
@@ -18,10 +18,7 @@
 #  This CMakeLists intended to be used in docs/EN
 #    Refactored for internationalization
 ###
-# Last Mod GP 12/2022
-#  Removed HPCC in a VM
-#  Removed Monitoring  
-#  Removed ECLWatch  
+# Last Mod GP 11/2023
```

Comment:
Did you mean to lose the comments about changes.  Should there be a comment saying "Removed AWS..."